platform/android/CHANGELOG.md

```diff
@@ -2,11 +2,28 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
-## master
+## 8.3.0-alpha.1 - July 31, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.2.0...android-v8.3.0-alpha.1) since [Mapbox Maps SDK for Android v8.1.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.1.0):
+
+### Features
+ - Do not include CJK ideographs in the offline packs by default [#14269](https://github.com/mapbox/mapbox-gl-native/pull/14269)
```

Comment:
Can we expand on this as well? We can mention that it decreases the size but changes the appearance and requires the (now enabled by default) local glyph generation?